### PR TITLE
Add Stack docs for Docker

### PR DESCRIPTION
**Pull request by hardillb**
I'll add the K8s doc as a whole block later

### CODE REVIEW

File: docs/install/docker/stacks.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+For container based deployment models, this covers three things:
+
+ - `memory` - the value to apply (in MB) to limit container to consuming.
+ - `cpu` - a value between 1 and 100 that is the % of a CPU core the container should be allowed to consume.
+ - `container` - this is the fully qualified name of the container to use. The default container built when following the install instructions is named `flowforge/node-red:latest`
+
+## Creating Containers
+
+There is an example `Dockerfile` and `package.json` in the `node-red-container` 
+directory. This will start with `nodered/node-red:latest` as it's base and then
```
Comment (knolleary) [inline docs/install/docker/stacks.md:15]: Keeping in mind these docs are in the core repo, which directory is this pointing at? If it's a file in a repo, can we link to it?

File: docs/install/docker/stacks.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+}
+```
+
+To build the container run the following:
+
+```shell
+docker build node-red-container -t flowforge/node-red-dashboard:2.2.2
+```
+
+You would then enter `flowforge/node-red-dashboard:2.2.2` in the containers section
```
Comment (knolleary) [inline docs/install/docker/stacks.md:54]:
```suggestion
You would then enter `flowforge/node-red-dashboard:2.2.2` in the `container` section
```

File: docs/install/docker/stacks.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+```shell
+docker build node-red-container -t flowforge/node-red-dashboard:2.2.2
+```
+
+You would then enter `flowforge/node-red-dashboard:2.2.2` in the containers section 
+of the Stack configuration.
+
+*Note:* with the 0.3 release, it is not possible to change the Stack being used
+by a project That will come in a future release.
```
Comment (knolleary) [inline docs/install/docker/stacks.md:58]:
```suggestion
*Note:* with the 0.3 release, it is not possible to change the Stack being used
by a project. That will come in a future release.
```

File: docs/install/docker/stacks.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# Docker Project Stacks
+
+A Project Stack defines a set of platform configuration options that will get
+applied to each project when created
```
Comment (knolleary) [inline docs/install/docker/stacks.md:4]:
```suggestion
applied to each project when created.
```

File: docs/install/docker/stacks.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+For container based deployment models, this covers three things:
+
+ - `memory` - the value to apply (in MB) to limit container to consuming.
+ - `cpu` - a value between 1 and 100 that is the % of a CPU core the container should be allowed to consume.
```
Comment (knolleary) [inline docs/install/docker/stacks.md:9]: Should we recommend a minimum?

File: docs/install/docker/stacks.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
+# Docker Project Stacks
+
+A Project Stack defines a set of platform configuration options that will get
+applied to each project when created
+
+For container based deployment models, this covers three things:
+
+ - `memory` - the value to apply (in MB) to limit container to consuming.
```
Comment (knolleary) [inline docs/install/docker/stacks.md:8]:
```suggestion
 - `memory` - the amount of memory (in MB) to limit container to. Recommended minimum: `256`.
```

File: docs/install/docker/stacks.md (hunk 1 above)
Comment (hardillb) [reply docs/install/docker/stacks.md:15]: Yep, we can link to https://github.com/flowforge/docker-compose/tree/main/node-red-container (the build.sh will get removed as it's leaked in from my local env)

**Pull request merged**